notebooks/peerscout/peerscout-recommend-reviewing-editors.ipynb

```diff
@@ -300,17 +300,26 @@
     ") -> Iterable[dict]:\n",
     "    for row in prediction_df.itertuples():\n",
     "        for predicted_editor in row.prediction:\n",
-    "            yield remove_key_with_null_value({\n",
-    "                'version_id': row.version_id,\n",
-    "                'score': predicted_editor[0],\n",
-    "                'name': predicted_editor[1],\n",
-    "                'person_id': editor_person_id_by_name_map[predicted_editor[1]],\n",
-    "                'matching_keyword_score': predicted_editor[2],\n",
-    "                'matching_keywords': [{\n",
-    "                    'score': keyword_score,\n",
-    "                    'keyword': keyword\n",
-    "                } for keyword_score, keyword in predicted_editor[3]],\n",
-    "            })\n"
+    "            df_prediction_filtered_by_version_id = (\n",
```

Comment:
just some naming for consistency, at least within the notebook `df` would be a suffix. In general I think I put the type indicator at the end.

something we discovered when we looked at it together, that DataFrame is containing exactly one version.

Not sure of a good name, perhaps even just: `this_manuscript_version_df` or `prediction_manuscript_version_df`.